cli/changelog/0.51.1.md

```diff
@@ -0,0 +1,9 @@
+### Features
+
+- Caching support for federated graphs (#1144)
```

Comment:
If this release has a new feature should it not be a minor bump?

Reply:
True, I bumped the version number before I looked at the commit log. Changing this to 0.52.0.

Reply:
Done and pushed, I hope I didn't forget any 0.51.1 anywhere. Double checking the diff.